static/src/App.vue

```diff
@@ -62,37 +24,18 @@ export default defineComponent({
     <i-layout-header>
       <h1 class="_text-align:center">Annoto</h1>
     </i-layout-header>
-    <i-layout-content>
-      <ProofOfCondition
-        v-model:isAttentive="isAttentive"
-        v-model:isTrained="isTrained"
-      />
-      <UserInformationLabel :username="user" />
-      <div v-if="imageId">
-        <ImageDisplay
-          id="image-display"
-          :task-id="taskId"
-          :image-id="imageId"
-        />
-        <i-button-group block>
-          <AnnotationButton
-            v-for="label in labels"
-            :id="'annotation-button-' + paramCase(label)"
-            :key="label"
-            :label="label"
-            username="user"
-            :is-attentive="isAttentive"
-            :is-trained="isTrained"
-            :competency="competency"
-            :task-id="taskId"
-            :image-id="imageId"
-            @annotation-saved="nextImage"
-          />
-        </i-button-group>
-      </div>
-      <i-card v-else id="no-more-images" class="margin-y:20px">
-        No more Images to label
-      </i-card>
-    </i-layout-content>
+    <LoginView
+      v-if="page == 'login'"
+      id="login-view"
+      :page="page"
```

Comment:
Is this page binding going anywhere? I cant find it in LoginView

Reply:
Well seen! It's an old leftover and should definitely be removed